### PR TITLE
UA parser: missed hydro production. Add capacities.

### PR DESCRIPTION
**Pull request by denplis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,6 +176,7 @@ Production capacities are centralized in the [capacities.json](https://github.co
   - Coal: [wikipedia.org](https://es.wikipedia.org/wiki/Central_t%C3%A9rmica_de_Es_Murterar)
 - Sweden: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
 - Switzerland: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
+- Ukraine [wikipedia.org] https://ru.wikipedia.org/wiki/%D0%AD%D0%BD%D0%B5%D1%80%D0%B3%D0%B5%D1%82%D0%B8%D0%BA%D0%B0_%D0%A3%D0%BA%D1%80%D0%B0%D0%B8%D0%BD%D1%8B
```
Comment (corradio) [inline README.md]: You forgot the parenthesis to actually turn it into a link

File: config/zones.json [hunk 2]
```diff
@@ -1146,6 +1146,13 @@
     ]
   },
   "UA": {
+    "capacity": {
+      "hydro": 5380.8,
+      "hydro storage": 1
```
Comment (corradio) [inline config/zones.json]: Missing comma

File: README.md (hunk 1 above)
Comment (denplis) [reply README.md]: >Can you confirm that the key consumptiongaespump gives hydro storage, with positive >values being amount of energy stored (and negative would be produced)?
consumptiongaespump - means, how much storage pumps are consuming energy, always positive.

And producing  we can see only in summarize parameter `gesgaes` which means
( all hydro + energy production by hydro storage)

**Comment by denplis** [comment]
@corradio 
`consumptiongaespump` - how much energy pump can consume, always positive value

Producing of hydro accumulation station reflected in `gesgaes` which is summarised ( all hydro + all hydro storage producing ).

**Pull request merged**